velox/vector/BaseVector.h

```diff
@@ -931,7 +931,7 @@ class BaseVector {
 };
 
 /// Loops over rows in 'ranges' and invokes 'func' for each row.
-/// @param TFunc A void function taking two arguments: targetIndex and
+/// @tparam TFunc A void function taking two arguments: targetIndex and
```

Comment:
Should be `@param func`?

Reply:
`tparam` means template argument, see: https://micro-os-plus.github.io/develop/doxygen-style-guide/

I can also change to `@param` if you'd like

Reply:
If you document `func` itself there is not need to doc `TFunc`